docs/source/deployment/heroku.md

```diff
@@ -39,11 +39,11 @@ server.listen({ port: process.env.PORT || 4000 }).then(({ url }) => {
 
 ## Deploying the project
 
-There are a couple of ways to push projects to Heroku. Automatically, with GitHub integration, or manually using Heroku push.
```

Comment:
`Heroku push` may be correct, referring to using the Heroku CLI:
https://devcenter.heroku.com/articles/heroku-cli-commands

(I'm not sure, but just throwing it out there)

Reply:
Weirdly, one thing the Heroku CLI _doesn't_ support is deployment (except for Docker containers), and you just use a barebones git push